apps/www/__registry__/default/example/combobox-form.tsx

```diff
@@ -1,19 +1,20 @@
-"use client"
+"use client";
 
-import { zodResolver } from "@hookform/resolvers/zod"
-import { Check, ChevronsUpDown } from "lucide-react"
-import { useForm } from "react-hook-form"
-import { z } from "zod"
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Check, ChevronsUpDown } from "lucide-react";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
 
-import { cn } from "@/lib/utils"
-import { Button } from "@/registry/default/ui/button"
+import { cn } from "@/lib/utils";
+import { Button } from "@/registry/default/ui/button";
```

Comment:
formatting should stay as is. If needed, a separate PR should be created so this isn't polluted.